lib/ASTGen/Sources/ASTGen/PluginHost.swift

```diff
@@ -13,6 +13,7 @@
 import CASTBridging
 import CBasicBridging
 import SwiftSyntax
+import swiftLLVMJSON
```

Comment:
Should we name ASTGen and LLVMJSON with `Swift` instead? ie. `SwiftASTGen`, `SwiftLLVMJSON`?

Reply:
Maybe we could make `EMIT_MODULE` a one_value_keywords accepting the module name, but at this point,  I don't really care about the name.